CHANGELOG.md

```diff
@@ -1,6 +1,53 @@
 # go-ipfs changelog
 
-## 0.4.16
+## 0.4.17  2018-07-20
+
+Ipfs 0.4.17 is a quick release to fix a major performance regression in bitswap.
+However, while motivated by that fix, it contains a few other goodies that will
+excite some users.
+
+The headline feature in this release is [urlstore][] support. Urlstore is a
+generalization of the filestore backend that can fetch file blocks from remote
+URLs on-demand instead of storing them in the local datastore.
+
+Users may also notice reduced CPU usage on IPFS nodes participating in an IPFS
+cluster.
```

Comment:
" due to .... "

Reply:
the fast repo/stat? yes it's used. But I think it reduces disk IO, not cpu..

Reply:
Ah. It reduces both, IIRC. However, in retrospect, that's probably more of a cluster release note.